docs/components/connectors/out-of-the-box-connectors/uipath.md

```diff
@@ -0,0 +1,256 @@
+      "x-xss-protection":"1",
+      "x-content-type-options":"nosniff",
+      "x-robots-tag":"noindex,nofollow",
+      "content-type":"application/json; odata.metadata=minimal; odata.streaming=true",
+      "connection":"keep-alive",
+      "cache-control":"no-cache, no-store, must-revalidate",
+      "odata-version":"4.0"
+   },
+   "body":{
+      "@odata.context":"https://cloud.uipath.com/MyOrg/MyTenant/orchestrator_/odata/$metadata#QueueItems",
+      "@odata.count":1,
+      "value":[
+         {
+            "QueueDefinitionId":165001,
+            "Encrypted":false,
+            "Status":"New",
+            "ReviewStatus":"None",
+            "Key":"2196eb07-c96a-4f47-a734-326dd5d58a9d",
+            "Reference":"test",
+            "Priority":"Low",
+            "DeferDate":"2023-01-12T00:00:00Z",
+            "SecondsInPreviousAttempts":0,
+            "RetryNumber":0,
+            "SpecificData":"{\"DynamicProperties\":{\"test\":\"test\"}}",
+            "CreationTime":"2023-01-20T10:13:20.66Z",
+            "RowVersion":"AAAAAE2f4GY=",
+            "OrganizationUnitId":1964413,
+            "OrganizationUnitFullyQualifiedName":"MyCorporateWorkspace",
+            "Id":436141352,
+            "SpecificContent":{
+               "test":"test"
+            }
+         }
+      ]
+   }
+}
+```
+
+## Appendix
+
+### Using UIPath Connector best practice
+
+There is no guarantee that a queue item will be processed right away. In that case, we suggest to utilize a BPMN polling loop pattern.
+
+![UIPath polling loop](../img/connectors-uipath-long-polling-pattern.png)
```

Comment:
@bastiankoerber, I have added this section but struggling to name a pattern. Calling it as `BPMN polling loop` but not sure whether it's technical enough. Please suggest.

<img width="1017" alt="Screenshot 2023-02-05 at 18 49 22" src="https://user-images.githubusercontent.com/108870003/216832828-2d8f6f36-e465-421b-9d82-bd58e59e19c3.png">

Reply:
Discussed with @bastiankoerber during daily. The term `BPMN polling loop` is good to use here.